components/ButtonNextPage/index.js

```diff
@@ -0,0 +1,12 @@
+import { StyledButtonNextPage } from "./styles";
+import { StyledContainer } from "./styles";
+
+export default function ButtonNextPage({ text }) {
+  return (
+    <>
+      <StyledContainer>
+        <StyledButtonNextPage type="type">{text}</StyledButtonNextPage>
+      </StyledContainer>
+    </>
+  );
+}
```

Comment:
This component is a bit over engineered.

- The component/structure is overly nested. If you examine the DOM, you'll notice that within your Link (or anchor element), there's a div element, inside of which you'll find a button element - much like a little Matryoshka 😁
-  In very simple terms, a clickable element is inside a clickable element. Please try to find a solution by using only the next/Link component. If you want it to look like a button, you could solve this task with styling.
- To make the structure clearer I would like to suggest you to do without a separate `ButtonNextPage ` component, not only because this component is used just once. Apart from that, the name of the component is a bit misleading.